### DISCUSSION TITLE
Active Sidebar Items Expands All Main Categories

### DISCUSSION THREAD

**Discussion by ChrisLeeThompson** [imfing/hextra#213] [Q&A] [answered]
Thank you for the updated sidebar with the option for multiple separators!

I have a question about the new sidebar behavior. It appears that when I click on a link in the sidebar all of the main sidebar categories will open. For example, if I collapse the Guide category, then select an item in the Advanced category, such as multi-language, then the Guide category expands.

If I have many top-level items in the sidebar, this behavior could expand them all and display too much information at once. Is it possible to change this behavior?

I have attempted to add "sidebar: open: false" to the page header, but this does not appear to have any effect.

**Comment by imfing** [comment, marked as answer]
> I have a question about the new sidebar behavior.

what do you mean by "new sidebar behavior", the behavior of the sidebar should remain the same as before.

> For example, if I collapse the Guide category, then select an item in the Advanced category, such as multi-language, then the Guide category expands.

that's expected because currently the sidebar is static and the state is not persisted. thus if you navigate between pages, the sidebar state will be reverted.

> If I have many top-level items in the sidebar, this behavior could expand them all and display too much information at once. Is it possible to change this behavior?

try reduce the number of sidebar items by putting them into sub-folder, or you can collapse that sidebar entry like the "Advanced" category

**Reply by ChrisLeeThompson** [reply]
Ah, yes, I was incorrect. I checked the previous version and it has the same expanding behavior.

I also found where to close the sidebar for the main categories, such as Guide. My apologies - I forgot how to do this. 

Thank you again for a great Hugo theme!
